backend/src/controllers/utils/index.ts

```diff
@@ -72,3 +81,186 @@ export const getTemplatesPath = (): string => {
+      })]
+    });
+    const accountingCustomerParty = new AccountingCustomerParty({
+      party: customerParty
+    });
+
+    const legalMonetaryTotal = new LegalMonetaryTotal({
+      lineExtensionAmount: new UdtAmount(savedInvoice.money.totalWithoutTax.toFixed(2), currencyID),
+      taxInclusiveAmount: new UdtAmount(savedInvoice.money.total.toFixed(2), currencyID),
+      taxExclusiveAmount: new UdtAmount(savedInvoice.money.totalWithoutTax.toFixed(2), currencyID),
+      payableAmount: new UdtAmount(savedInvoice.money.total.toFixed(2), currencyID)
+    });
+
+
+    let taxObject: { id: string | UdtIdentifier; percent: string | UdtPercent; taxScheme: TaxScheme | undefined; taxExemptionReasonCode?: string | undefined };
+    let classifiedTaxObject: { id: string | UdtIdentifier; percent: string | UdtPercent; taxScheme: TaxScheme | undefined };
+    if (savedInvoice.client.country !== DEFAULT_COUNTRY_CODE) {
+      taxObject = {
+        id: 'AE',
+        percent: '0',
+        taxExemptionReasonCode: 'VATEX-EU-AE',
+        taxScheme: taxSchemeIDVAT
+      }
+      classifiedTaxObject = {
+        id: 'AE',
+        percent: '0',
+        taxScheme: taxSchemeIDVAT
+      }
+    } else {
+      taxObject = {
+        id: 'S',
+        percent: '21',
+        taxScheme: taxSchemeIDVAT
+      }
+      classifiedTaxObject = taxObject;
+    }
+
+
+    const taxTotal = new TaxTotal({
+      taxAmount: new UdtAmount(savedInvoice.money.totalTax.toFixed(2), currencyID),
+      taxSubtotals: [new TaxSubtotal({
+        taxableAmount: new UdtAmount(savedInvoice.money.totalWithoutTax.toFixed(2), currencyID),
+        taxAmount: new UdtAmount(savedInvoice.money.totalTax.toFixed(2), currencyID),
+        taxCategory: new TaxCategory(taxObject)
+      })]
+    });
+
+    const paymentMeans = new PaymentMeans({
+      paymentMeansCode: '31',
+      payeeFinancialAccount: new PayeeFinancialAccount({
+        id: companyConfig.company.iban,
+        financialInstitutioBranch: new FinancialInstitutionBranch({
+          id: companyConfig.company.bic
+        })
+      })
+    });
+
+    const orderRef = savedInvoice.orderNr && savedInvoice.orderNr.trim().length !== 0 ? savedInvoice.orderNr : savedInvoice._id;
+
+    invoiceXml.addProperty('xmlns', 'urn:oasis:names:specification:ubl:schema:xsd:Invoice-2');
+    invoiceXml.addProperty('xmlns:cac', 'urn:oasis:names:specification:ubl:schema:xsd:CommonAggregateComponents-2');
+    invoiceXml.addProperty('xmlns:cbc', 'urn:oasis:names:specification:ubl:schema:xsd:CommonBasicComponents-2');
+    invoiceXml.setCustomizationID('urn:cen.eu:en16931:2017#compliant#urn:fdc:peppol.eu:2017:poacc:billing:3.0');
+    invoiceXml.setProfileID('urn:fdc:peppol.eu:2017:poacc:billing:01:1.0');
+    invoiceXml.setIssueDate(moment(savedInvoice.date).format('YYYY-MM-DD'));
+    invoiceXml.setDueDate(moment(savedInvoice.date).add(30, 'days').format('YYYY-MM-DD'));
+    invoiceXml.setInvoiceTypeCode('380');
```

Comment:
Put the 380 in a constant
For these constants, maybe you could even add a little JSDoc with the link to further details (the webpages you showed me)

```js
/** URL TO DOCS */
const NormalInvoice = '380';
```

(or something)